.golangci.yaml

```diff
@@ -77,9 +81,6 @@ linters:
     - unused
     - whitespace
 
-service:
```

Comment:
This configuration does not exist anymore
https://golangci-lint.run/usage/configuration/